sharepoint/sharepoint-ps/sharepoint-online/Add-SPOSiteScriptPackage.md

```diff
@@ -0,0 +1,102 @@
+---
+external help file: Microsoft.Online.SharePoint.PowerShell.dll-Help.xml
+Module Name: Microsoft.Online.SharePoint.PowerShell
+online version:
+schema: 2.0.0
+---
+
+# Add-SPOSiteScriptPackage
+
+## SYNOPSIS
+Uploads a new site script package for use either directly or in a site design.
+
+## SYNTAX
+
+```
+Add-SPOSiteScriptPackage -Title <String> -ContentPath <String> [-Description <String>] [<CommonParameters>]
+```
+
+## DESCRIPTION
+Uploads a new site script package for use either directly or in a site design. The package file must be a zip archive containing all the files necessary for the site script. A file called “manifest.json” with script actions must be present in this zip file.
+
+## EXAMPLES
+
+### Example
+This example adds a site script package as a zip file containing a manifest.json script, as well as a Dataverse solution zip file with a Power Automate flow definition. The content of manifest.json is as follows:
```

Comment:
please move the description at the end of the example

Reply:
@get-itips Done, thanks.